funcake_villanova_dag.py

```diff
@@ -85,6 +88,21 @@
     dag=VILLANOVA_HARVEST_DAG
 )
 
+TRANSFORM = BashOperator(
```

Comment:
Is there going to be another transform function for schematron?  If so, could we name this method XSLT_TRANSFORM so that it matches the task id and is more explicit when there are multiple transform functions?

Reply:
@cmharlow @bibliotechy ☝️